src/applications/personalization/dashboard/covid-19.jsx

```diff
@@ -0,0 +1,142 @@
+  { name: 'Marianna VA Clinic', id: '573GK' },
+  { name: 'Mayaguez VA Clinic', id: '672BZ' },
+  { name: 'Miami Flagler VA Clinic', id: '000' },
+  { name: 'Miami VA Healthcare System', id: '546B0' },
+  { name: 'Minneapolis VA Medical Center', id: '618' },
+  { name: 'Montevideo VA Clinic', id: '656GB' },
+  { name: 'Naples VA Clinic', id: '516GF' },
+  { name: 'New Port Richey VA Clinic', id: '673BZ' },
+  { name: 'North Florida/South Georgia Veterans Health System', id: '000' },
+  { name: 'Northwest Metro VA Clinic', id: '618GI' },
+  { name: 'Ocala VA Clinic', id: '573GD' },
+  { name: 'Okeechobee VA Clinic', id: '548GF' },
+  { name: 'Orlando VA Healthcare System', id: '675' },
+  { name: 'Palatka VA Clinic', id: '573GL' },
+  { name: 'Palm Harbor VA Clinic', id: '516GC' },
+  { name: 'Pembroke Pines VA Clinic', id: '546GD' },
+  { name: 'Perry VA Clinic', id: '573GN' },
+  { name: 'Port Charlotte VA Clinic', id: '516GE' },
+  { name: 'Rice Lake VA Clinic', id: '618GM' },
+  { name: 'Rochester VA Clinic', id: '618GG' },
+  { name: 'Saint Augustine VA Clinic', id: '573GE' },
+  { name: 'Saint Croix VA Clinic', id: '672GA' },
+  { name: 'Saint Thomas VA Clinic', id: '672GB' },
+  { name: 'Sarasota VA Clinic', id: '516GA' },
+  { name: 'Sebring VA Clinic', id: '516GH' },
+  { name: 'Southpoint Clinic', id: '000' },
+  {
+    name: 'St Lucie County PTSD Clinical Team (PCT) Outpatient Program',
+    id: '000',
+  },
+  { name: 'St. Cloud VA Medical Center', id: '656' },
+  { name: 'St. Petersburg VA Clinic', id: '516GB' },
+  { name: 'Stuart VA Clinic', id: '548GC' },
+  { name: 'Tavares VA Clinic', id: '675GE' },
+  { name: 'The Villages VA Clinic', id: '573GI' },
+  { name: 'Twin Ports VA Clinic', id: '618BY' },
+  { name: 'Utuado VA Clinic', id: '000' },
+  { name: 'VA Caribbean Healthcare System', id: '000' },
+  { name: 'VA Moore Haven Outpatient Clinic', id: '000' },
+  { name: 'Valdosta VA Clinic', id: '573GA' },
+  { name: 'Vero Beach VA Clinic', id: '548GE' },
+  { name: 'Vieques Rural Outpatient Clinic', id: '000' },
+  { name: 'Viera VA Clinic', id: '675GA' },
+  { name: 'Viera VA Clinic', id: '675GA' },
+  { name: 'West Palm Beach VA Medical Center', id: '548' },
+  { name: 'Zephyrhills VA Clinic', id: '673GF' },
+  { name: "James A. Haley Veterans' Hospital Primary Care Annex", id: '000' },
+  { name: "Sergeant Ernest I. 'Boots' Thomas VA Clinic", id: '573GF' },
+  { name: "St. Mary's VA Clinic", id: '573GJ' },
+  { name: "William 'Bill' Kling VA Outpatient Clinic", id: '546BZ' },
+  { name: "William V. Chappell, Jr. Veterans' Outpatient Clinic", id: '675GB' },
+];
+
+/**
+ * This filters the full list of eligible facilities and returns just a set of
+ * the three-digit health system IDs that appear in that list.
+ */
+export const eligibleHealthSystems = new Set(
+  // the health system is always the first three digits of a facility ID
+  eligibleFacilities.map(facility => facility.id.substring(0, 3)),
+);
```

Comment:
I'm wondering if this and the selector [above](https://github.com/department-of-veterans-affairs/vets-website/pull/11952/files#diff-b2862df01b28b69f7943d0beca5dc1eaR377) could be chunked somewhere it can be shared with VAOS (if they're doing something similar)?

Reply:
Good thought. But I don't think they care about this special list of facilities that offer health chat. And this is really fuzzy imprecise logic that we shouldn't promote reuse of, IMO, since it's not 100% accurate.